manifests/topic.pp

```diff
@@ -32,17 +35,29 @@
 #   See the Kafka documentation for full details on the topic configs.
 #
 define kafka::topic (
-  String[1] $ensure                           = '',
-  String[1] $zookeeper                        = '',
+  String[1] $ensure                           = undef,
```

Comment:
This needs to wrapped into `Optional` since it's defaulting to `undef` now
Also this could be narrowed down to an Enum. e.g:

```suggestion
Optional[Enum['present', 'absent']] $ensure   = undef
```

Although absent won't remove the topic since there's not functionality for that :smile: